lib/env-bundler.js

```diff
@@ -630,102 +636,101 @@ const resourceExtractorByType = {
   },
 };
 
-const processContentTypeAttribute = function(attribute) {
+/* const processContentTypeAttribute = function (attribute) {
```

Comment:
I think it can be removed, no?

Reply:
Yes, this was marked as unused by the linter. I commented it out instead of deleting it because I didn't know if it could be useful. But probably you are right, we can just remove it.

Reply:
done